discern.lic

```diff
@@ -0,0 +1,46 @@
+custom_require.call(%w[common common-travel events spellmonitor drinfomon])
+
+class AutoDiscern
+    include DRC
+    include DRCA
+
+    def initialize()
+    settings = get_settings
+    if(settings.force_discerns)
+        UserVars.discerns = {}
+    end
+    echo 'Reading spell list'
+    buffspelllist = settings.buff_spells
+    offspelllist =  settings.offensive_spells
+    ctspelllist = settings.training_spells

```

Comment:
Do you need to store each of these spell lists into a local var? What do you think about something like `settings.training_spells.each {|x| ...}` instead?